### ISSUE TITLE
Templated Return Types (explicit instantiation)

### ISSUE THREAD

**Issue by peaceshi** [jeff-hykin/cpp-textmate-grammar#79]
https://en.cppreference.com/w/cpp/language/function_template
https://docs.microsoft.com/en-us/cpp/cpp/explicit-instantiation?view=vs-2019

**Comment by matter123** [comment]
Blocked by #80

**Comment by jeff-hykin** [comment]
#184 adds return types (which is one step closer to fixing this)

**Comment by jeff-hykin** [comment]
closing in favor of #246
